hooks/command

```diff
@@ -0,0 +1,307 @@
+#!/bin/bash
+
+set -euo pipefail
+
+DIR="$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd)"
+
+# shellcheck source=lib/plugin.bash
+. "$DIR/../lib/plugin.bash"
+
+# Check if required tools are installed
+if ! command -v helm &> /dev/null; then
+  echo "❌ Error: helm is not installed or not in PATH"
+  exit 1
+fi
+
+if ! command -v jq &> /dev/null; then
+  echo "❌ Error: jq is not installed or not in PATH"
+  exit 1
+fi
+
+# Read configuration
+MODE=$(plugin_read_config MODE "")
+CHART=$(plugin_read_config CHART "")
+RELEASE=$(plugin_read_config RELEASE "")
+NAMESPACE=$(plugin_read_config NAMESPACE "default")
+KUBECONFIG=$(plugin_read_config KUBECONFIG "")
+TIMEOUT=$(plugin_read_config TIMEOUT "300s")
+WAIT=$(plugin_read_config WAIT "true")
+ATOMIC=$(plugin_read_config ATOMIC "true")
+CREATE_NAMESPACE=$(plugin_read_config CREATE_NAMESPACE "false")
+DRY_RUN=$(plugin_read_config DRY_RUN "false")
+REVISION=$(plugin_read_config REVISION "")
+REPO_URL=$(plugin_read_config REPO_URL "")
+REPO_NAME=$(plugin_read_config REPO_NAME "")
+FORCE=$(plugin_read_config FORCE "false")
+HISTORY_MAX=$(plugin_read_config HISTORY_MAX "10")
+
+# Validate required configuration
+if [ -z "${MODE}" ]; then
+  echo "❌ Error: 'mode' is required (deploy or rollback)"
+  exit 1
+fi
+
+if [ -z "${RELEASE}" ]; then
+  echo "❌ Error: 'release' is required"
+  exit 1
+fi
+
+if [ "${MODE}" = "deploy" ] && [ -z "${CHART}" ]; then
+  echo "❌ Error: 'chart' is required for deploy mode"
+  exit 1
+fi
+
+if [ "${MODE}" != "deploy" ] && [ "${MODE}" != "rollback" ]; then
+  echo "❌ Error: 'mode' must be either 'deploy' or 'rollback'"
+  exit 1
+fi
+
+# Set kubeconfig if provided
+if [ -n "${KUBECONFIG}" ]; then
+  export KUBECONFIG="${KUBECONFIG}"
+  echo "Using KUBECONFIG environment variable: ${KUBECONFIG}"
+else
+  echo "No KUBECONFIG environment variable set"
+fi
+
+# Debug kubeconfig file existence
+echo "Checking kubeconfig file: ${KUBECONFIG}"
+if [ -f "${KUBECONFIG}" ]; then
+  echo "✅ Kubeconfig file exists at: ${KUBECONFIG}"
+  ls -la "${KUBECONFIG}"
+else
+  echo "❌ Kubeconfig file not found at: ${KUBECONFIG}"
+  echo "Current working directory: $(pwd)"
+  echo "Available files:"
+  find . -name "*config*" -type f 2>/dev/null || echo "No config files found"
+fi
```

Comment:
Are these checks necessary when all subsequent `helm` commands perform a `if [ -n "${KUBECONFIG}" ]` check and conditionally append `--kubeconfig ${KUBECONFIG}` to the commands if set?